src/pyramid_debugtoolbar/panels/templates/session.dbtmako

```diff
@@ -0,0 +1,90 @@
+<h5>Session Configuration</h5>
+% if configuration:
+	<p>Using `ISessionFactory`: <code>${configuration}</code></p>
```

Comment:
Using backticks has no effect on formatting. I'd prefer either their removal throughout or use of `<code>`. Also `<pre>` is not a good option because its default display is a block, not inline.